test/extensions/transport_sockets/tls/context_impl_test.cc

```diff
@@ -1971,92 +1971,30 @@ TEST_F(ServerContextConfigImplTest, PrivateKeyMethodLoadFailureNoProvider) {
       "Failed to load private key provider: mock_provider");
 }
 
-TEST_F(ServerContextConfigImplTest, PrivateKeyMethodLoadFailureNoMethod) {
```

Comment:
We should remove this old test, since the old `private_key_provider` is still available for the user, we should keep the old tests.

Reply:
done, thanks!